cpu/log_generator.py

```diff
@@ -81,6 +68,20 @@ def test_sched_scoreboard_setup(self):
                 (self.sched_scoreboard_dir)
             process.run(cmd, ignore_status=True, sudo=True, shell=True)
 
+    def test_sched_scoreboard_setup(self):
+        """_summary_
+        Here in this test case we are cloning the sched_scoreboard
+        repo and prepareing the sched_scoreboard to run.
+        """
+        scoreboard_setup_log_file = self.logdir + "/sched_scoreboard_setup"
+        os.mkdir(scoreboard_setup_log_file)
+
+        log_file = "%s/sched_scoreboard_command_output_%s.log" % (
+            scoreboard_setup_log_file, self.kernel_ver)
+        cmd = 'echo "Using command: %s >> %s" 2>&1' % (
+            self.user_command, log_file)
+        process.run(cmd, ignore_status=True, sudo=True, shell=True)
+
         cmd = 'lscpu >> "%s" 2>&1' % (log_file)
```

Comment:
This test should not need to run `lscpu`
This should get captured by **sysinfo** utility (preferably one time only)